draft-quic-atsss-reqs.mkd

```diff
@@ -118,7 +118,34 @@ The two scenarios that we consider in this document are:
 - the failure of one access network that forces all data to switch to the other access network
 - the possibility of splitting the transmission of the data over the two access networks even if they have different characteristics (delay, bandwidth, congestion)
 
+## MPTCP-Proxied Connections
 
+This section describes how TCP connections matching ATSSS rules are currently established over different access networks.
+
+An ATSSS-capable UE can make use of the MPTCP functionality by establishing MPTCP-assisted connections via the MPTCP proxy relying upon the Convert Protocol {{I-D.ietf-tcpm-converter}}. The UE behaves as a “Client”, while the MPTCP proxy behaves as a Transport Converter {{I-D.ietf-tcpm-converter}}. 
+
+The ATSSS-capable UE sends packets bound to connections matching an ATSSS rule to the provisioned Transport Converter and destination port number. Concretely, the UE initiates the MPTCP connection towards the Transport Converter and indicates the IP address and port number of the Server within the connection establishment packet (i.e., in the payload of the SYN sent to the Transport Converter).  Doing so enables the Transport Converter to immediately initiate a connection towards that Server, without experiencing an extra delay. The Transport Converter waits until the receipt of the confirmation that the Server agrees to establish the connection before confirming it to the Client.
+
+A flow example of an MPTCP-proxied connection is show in {{fig-example}}. This example assumes the Server is not MPTCP-aware. The access network used to establish the matching ATSSS rule to select the access network to be used to establish the first subflow. The instruction included in the ATSSS rule will be followed for the management of the MPTCP connection. 
```

Comment:
> The access network used to establish the matching ATSSS rule to select the access network to be used to establish the first subflow.

This sentence is not correct.

Reply:
> The instruction included in the ATSSS rule will be followed for the management of the MPTCP connection.

I am new to the ATSSS spec: Is an ATSSS rule equivalent to a particular steering mode ?

Reply:
"Steering Mode" is a mandatory attribute of an ATSSS rule.

Reply:
> 
> 
> > The access network used to establish the matching ATSSS rule to select the access network to be used to establish the first subflow.
> 
> This sentence is not correct.

Fixed in d2d4c90